client/db_test.go

```diff
@@ -344,26 +344,31 @@ func TestCommonMethods(t *testing.T) {
 		method string
 	}
 	blacklist := map[key]struct{}{
-		key{batchType, "InternalAddCall"}:   {},
-		key{dbType, "AdminMerge"}:           {},
-		key{dbType, "AdminSplit"}:           {},
-		key{dbType, "NewBatch"}:             {},
-		key{dbType, "Run"}:                  {},
-		key{dbType, "Txn"}:                  {},
-		key{dbType, "GetSender"}:            {},
-		key{txnType, "Commit"}:              {},
-		key{txnType, "CommitInBatch"}:       {},
-		key{txnType, "CommitNoCleanup"}:     {},
-		key{txnType, "Rollback"}:            {},
-		key{txnType, "Cleanup"}:             {},
-		key{txnType, "DebugName"}:           {},
-		key{txnType, "InternalSetPriority"}: {},
-		key{txnType, "NewBatch"}:            {},
-		key{txnType, "Run"}:                 {},
-		key{txnType, "SetDebugName"}:        {},
-		key{txnType, "SetIsolation"}:        {},
-		key{txnType, "SetSystemDBTrigger"}:  {},
-		key{txnType, "SystemDBTrigger"}:     {},
+		key{batchType, "InternalAddRequest"}:      {},
+		key{dbType, "AdminMerge"}:                 {},
+		key{dbType, "AdminSplit"}:                 {},
+		key{dbType, "GetProto"}:                   {}, // TODO(tschottdorf)
```

Comment:
I've removed `GetProto` from `batch` (it was unused, but relied on the `Post` field in `proto.Call` which now has no counterpart). This is described in the commit message, but I'll add a comment.